deploy/stacks/pipeline.py

```diff
@@ -742,6 +744,14 @@ def set_cloudfront_stage(self, target_env):
                     f'export enable_cw_rum={target_env.get("enable_cw_rum", False)}',
                     f'export resource_prefix={self.resource_prefix}',
                     f'export reauth_ttl={str(target_env.get("reauth_config", {}).get("ttl", 5))}',
+                    f'export custom_auth_provider={str(target_env.get("custom_auth", {}).get("provider", "None"))}',
```

Comment:
do we not need to match the `export` statements in these commands with the same in the `set_albfront_stage()` and also only create the UserGuide if custom_auth is not present?

Reply:
Thanks for noticing that. Added those exports as in the albfront stage